backend/apps/events/tests.py

```
@@ -0,0 +1,919 @@
+        payload = {
+            'title': 'New Internal',
+            'description': 'desc',
+            'event_type': 'internal',
+            'start_time': (self.now + timedelta(hours=3)).isoformat(),
+            'end_time': (self.now + timedelta(hours=4)).isoformat(),
+            'space_id': self.space.id,
+            'max_participants': 5
+        }
+        req = self.setup_request(self.rf.post('/', data=json.dumps(payload), content_type='application/json'))
+        res = EventListView.as_view()(req)
+        if res.status_code != 201: print("DEBUG:", res.content.decode('utf-8'))
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)